scripts/i18n/sync.js

```diff
@@ -126,7 +181,33 @@ async function syncTranslationRepo(code) {
   shell.exec(`git remote add source ${sourceRepoGitUrl}`)
   shell.exec(`git fetch source master`)
 
-  // TODO don't run the sync script if there is a current PR from the bot
+  const repository = await getRepository(owner, transRepoName)
+
+  if (repository.syncPullRequests.nodes.length > 0) {
+    logger.info(
+      `There are currently open sync pull requests. Please ask the language maintainers to merge the existing PR(s) in before opening another one. Exiting...`
+    )
+    process.exit(0)
```

Comment:
Since this is a failure it should exist non-zero (`process.exit(1)`)

Reply:
It's not a failure though—it just represents another possible branch in the state of the world and exits early. Like in a dependency update script, it's not a "failure" if there are no dependencies to be updated and the script exits early.